generators/server/templates/src/main/docker/_app.yml

```diff
@@ -55,7 +55,7 @@ services:
             - SPRING_CLOUD_CONSUL_PORT=8500
                   <%_ } _%>
                   <%_ if (serviceDiscoveryType == 'eureka') { _%>
-            - SPRING_CLOUD_CONFIG_URI=http://admin:admin@registry:8761/config
+            - SPRING_CLOUD_CONFIG_URI=http://admin:$${jhipster.registry.password}@registry:8761/config
```

Comment:
This is to bypass docker-compose env var substitution : https://docs.docker.com/compose/compose-file/#/variable-substitution (read carefully as it is not straightforward 😄 )
I actually spent quite a bit of time to understand why it didn't work with just one $.